akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -169,7 +168,7 @@ If using SSL then `tcp-tls` needs to be enabled and setup. See @ref[Artery docs
 for how to do this.
 
 
-#### Migration from 2.5.x Artery to 2.6.x Artery
+### Migration from 2.5.x Artery to 2.6.x Artery
```

Comment:
note that there were some structural changes in https://github.com/akka/akka/pull/27025
is this still correct?